shared/src/transport/http.rs

```diff
@@ -46,6 +47,14 @@ impl HttpTransport {
     }
 }
 
+impl Debug for HttpTransport {
```

Comment:
Good stuff, I think this is a more appropriate `Debug` implementation than the default one.

Reply:
I actually noticed an ignored test for a debug implementation in model/second/order.rs yesterday that causes us to lose coverage there. Maybe if it were as pretty as this we could remove the ignore there.